app/lib/infrastructure/tts_repository.dart

```diff
@@ -0,0 +1,114 @@
+part of 'package:word_link/domain/controllers/tts_controller.dart';
+
+class _TtsRepository extends TtsController {
+  final FlutterTts _flutterTts = FlutterTts();
+  TtsState _ttsState = TtsState.stopped;
+  Function(TtsState)? _onStateChange;
+  bool _isInitialized = false;
+
+  @override
+  TtsState get state => _ttsState;
+
+  FlutterTts get tts => _flutterTts;
+
+  @override
+  Future<void> initialize() async {
+    if (_isInitialized) return;
```

Comment:
This method will always get called once when the program is starting so the logic is redundant.
Remove this and change to is supported, and make it false only for Linux.